dashboard/app/models/pd/application/application_base.rb

```diff
@@ -35,9 +35,10 @@ class ApplicationBase < ActiveRecord::Base
     include ApplicationConstants
     include Pd::Form
 
-    after_initialize -> {self.status = :unreviewed}
+    after_initialize -> {self.status = :unreviewed}, if: :new_record?
     after_initialize :set_type_and_year
     before_validation :set_type_and_year
+
```

Comment:
nit: why this empty line. It's not a big deal, but I would prefer the `set_type_and_year` method to be grouped with the validations that refer to it. If you want to add a newline, do it after line 38.